{{cookiecutter.github_repository}}/provisioner/roles/postgresql/tasks/main.yml

```diff
@@ -11,20 +11,17 @@
   shell: wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
```

Comment:
Can we use https://docs.ansible.com/ansible/2.5/modules/apt_key_module.html here?